pkg/cmd/tknpac/resolve/resolve.go

```diff
@@ -63,23 +65,37 @@ func Command(run *params.Run, streams *cli.IOStreams) *cobra.Command {
 		Short: "Embed PipelineRun references as a single resource.",
 		RunE: func(cmd *cobra.Command, args []string) error {
 			ctx := context.Background()
-			err := run.Clients.NewClients(ctx, &run.Info)
+
+			errc := run.Clients.NewClients(ctx, &run.Info)
```

Comment:
Any reason for not checking `errc` after this call :thinking:

Reply:
we do it here: https://github.com/openshift-pipelines/pipelines-as-code/pull/1040/files/bf5cb5da59e3e0d7f21ef376ada1280672d598fc#diff-4ba4b7421ccbce2260b7d8f0ba91c39f10724f8bfdf16dba313cfd0a84515803R85

we don't want to error out if user don't have access on cluser....